authors/apps/articles/tests/test_article_model.py

```diff
@@ -15,3 +16,29 @@ def test_str_return_method_in_article_model(self):
             body='In water',
             author=Profile.objects.get(user=user.id))
         self.assertEqual(str(article), 'whale')
+
+    def test_get_all_tags_model_helper_if_no_article_exists(self):
+        response = model_helpers.get_all_available_tags()
+        self.assertIsNone(response)
+
+    """ Intergration tests"""
+    def test_get_all_tags_model_helper_if_author_tags_article(self):
+        user = self.activated_user()
+        article = Article.objects.create(
```

Comment:
I think we should consider moving some of this data to the `test_data` module. Manually putting the fields here makes it hard to test and creates duplication.

Reply:
Thanks for that I'll do it right away.